### PR TITLE
Rails gem is available

### PR DESCRIPTION
**Pull request by bodrovis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,26 @@ jQuery(function($) {
...
+
+And then execute:
+
+    $ bundle
+
+In your `application.js` you will need to add this line:
+
+    //= require jquery.instagram
+
+NOTE: this is a jQuery plugin so you will also need the `jquery-rails` gem
```
Comment (potomak) [inline README.md]: You could simply add a dependency to `jquery-rails` gem

Comment (bodrovis) [inline README.md]: Well, that is true, but someone could use different version of jquery-rails in his project (other from the one that i will specify here). Also many developers can have jQuery included in another way (from CDN for example). So this note is just a reminder to include jQuery and by no means it is required to use gem.

Comment (potomak) [inline README.md]: Ok, so please change it to tell that only jQuery is required and not specificaly `jquery-rails`.

**Comment by bodrovis** [comment]
Changed the Readme

File: README.md [hunk 2]
```diff
@@ -32,6 +32,25 @@ jQuery(function($) {
...
+
+And then execute:
+
+    $ bundle
+
+In your `application.js` you will need to add this line:
+
+    //= require jquery.instagram
+
+NOTE: this is a jQuery plugin so you will also need jQuery library.
```
Comment (potomak) [inline README.md]: Ok, now it's better, but it seems a bit obvious that a jquery plugin requires jquery, please just remove this note and I'll merge this PR right now, thanks.

**Comment by bodrovis** [comment]
Done

**Pull request merged**